packages/db/src/schema.ts

```diff
@@ -1,26 +1,4 @@
-import * as fse from "fs-extra";
-import path from "path";
+import { definitions } from "./definitions";
+import { forDefinitions } from "./graphql";
 
-import { makeExecutableSchema } from "@gnd/graphql-tools";
-
-export function readSchema() {
-  const schemaFile = path.join(__dirname, "schema.graphql");
-  const typeDefs = fse.readFileSync(schemaFile).toString();
-
-  return makeExecutableSchema({
-    typeDefs,
-    resolvers: {
-      Named: {
-        __resolveType: obj => {
-          if (obj.networkId) {
-            return "Network";
-          } else {
-            return "Contract";
-          }
-        }
-      }
-    }
-  });
-}
-
-export const schema = readSchema();
+export const schema = forDefinitions(definitions);
```

Comment:
Isn't this the exact same pattern from `db/src/connect`? Perhaps could be combined into one file somehow?

Reply:
It's the same pattern, but it comes from two different places. I prefer having them be separate modules - `connect` gets you the pouch stuff, `schema` gets you the graphql stuff

Reply:
That makes sense!